### PR TITLE
2023: FOSDEM: add cfp for ARM SB 3rd edition

### PR DESCRIPTION
**Pull request by TomaszAIR**

### CODE REVIEW

**Comment by TomaszAIR** [comment]
@pietrushnic please review this, basically the cfp is similar to the ones we used in the past

**Comment by TomaszAIR** [comment]
@pietrushnic could you please look at this?

File: 2023/FOSDEM/overview-of-sb-arm-3rd.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Title
+
+Overview of Secure Boot state in the ARM-based SoCs: 3rd edition
+
+# Abstract
+
+In the ARM world, Secure Boot is typically a BootROM feature, which allows for
+verification of the loaded binaries (firmware, bootloader, Linux kernel) before
+executing it. The main idea is to prevent untrusted code from running on our
```
Comment (pietrushnic) [inline 2023/FOSDEM/overview-of-sb-arm-3rd.md:9]:
```suggestion
executing it. The main idea is to prevent aunauthorized code from running on our
```

**Comment by TomaszAIR** [comment]
@pietrushnic thanks, merging this one.

**Pull request merged**